Dfe.Academies.Academisation.Service/Mappers/Legacy/ApplicationAggregate/LegacyApplicationServiceModelMapper.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Dfe.Academies.Academisation.Domain.Core.ApplicationAggregate;
+using Dfe.Academies.Academisation.IDomain.ApplicationAggregate;
+using Dfe.Academies.Academisation.IService.ServiceModels.Legacy.ApplicationAggregate;
+
+namespace Dfe.Academies.Academisation.Service.Mappers.Legacy.ApplicationAggregate;
+
+internal class LegacyApplicationServiceModelMapper
+{
+	internal static LegacyApplicationServiceModel MapToLegacyServiceModel(IApplication application)
+	{
+		LegacyApplicationServiceModel serviceModel = new(
+			LegacyKeyPersonServiceModelMapper.MapToServiceModels(application),
+			LegacySchoolServiceModelMapper.MapToServiceModels(application)
+			)
+		{
+			ApplicationId = application.ApplicationId.ToString(),
+
+			// For now this legacy service model only supports JoinAMat
+			ApplicationType = MapType(application),
+			ApplicationSubmitted = application.ApplicationStatus == ApplicationStatus.Submitted,
+
+			// These don't seem to be set in the Azure Data Factory pipeline
+			ApplicationStatusId = null,
+
+			// ToDo: ApplicationReference
+			////Name = application.ApplicationReference,
+		};
+
+		// How do we determine the Lead Author, we have a list of contributors?
+		IContributor? leadAuthor = application.Contributors.FirstOrDefault();
```

Comment:
Is the answer to this in the TRAMS api repo?

Reply:
I think it's probably in the A2C-SIP (A2B External old) repo. Contributors is one area where we started making some changes before deciding to go with the as-is rewrite. I _think_ that the Lead Author is probably just the person that created the application but I'm not certain.

Reply:
If we don't match the logic that was there before then there's a very good chance that when we move over to this API we will introduce issues. Just highlighting that as a risk I suppose.

Reply:
Yeah, I think contributors might be awkward - it doesn't seem to be implemented in the existing code. There might be a bit of analysis needed here. I'll bring it up in stand-up tomorrow.